operator/pkg/controller-runtime/cell.go

```diff
@@ -77,6 +78,7 @@ func newManager(params managerParams) (ctrlRuntime.Manager, error) {
 		Metrics: metricsserver.Options{
 			BindAddress: "0",
 		},
+		Cache: cache.Options{},
```

Comment:
any reason to set Cache as empty option here ? Thanks

Reply:
Good catch, I didn't man to include it. I've pushed a new revision that removes it and addresses the CI lint failure